### ISSUE TITLE
how to change path on the fly by code?

### ISSUE THREAD

**Issue by shaulnaim** [ngx-lottie/ngx-lottie#56]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ x] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## current behavior
if i have a lottie and i want to replace the animation on the fly according to events or notifications (i got 10 lotties) .i have to create multiple ng-lottie components and configs and replace using ngif or ngswitch
its a bummer.. :(
## Expected behavior
 i wish i could call config.path= newJson and with angular binding it will be replaced on the fly

can it be?

## Environment

<pre><code>
Libs:
- @angular/core version: X.Y.Z
- ngx-lottie version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by arturovt** [comment]
Yes, it's possible :)

Would you want to change the reference to `options` and animation will be re-created under the hood? Something like that:
```ts
@Component({
  template: '<ngx-lottie [options]="options">...'
})
class SomeComponent {
  options = {
    path: '/path-animation.json'
  }

  updateAnimation() {
    this.options = {
      path: '/path-to-NEW-animation.json'
    }
  }
}
```

I'm actually thinking that tracking object properties is performance-sensitive and might cause unwanted side effects:
```ts
❌❌
updateAnimation() {
  this.options.path = '/path-to-NEW-animation.json'
}

✔️✔️
updateAnimation() {
  this.options = {
    path: '/path-to-NEW-animation.json'
  }
}
```

This will behave closely to `OnPush` components which are checked and updated if any of the bindings are changed.

Wtyd?

**Comment by JohannesArtmeier** [comment]
This would approach be really awesome!

**Comment by marius-orha** [comment]
It's a really needed feature.

**Comment by LuigiMitrica** [comment]
This would help a lot!

**Comment by aakki** [comment]
It saves a lot of time, a great approach!!

**Comment by arturovt** [comment]
I have published `ngx-lottie@6.1.0`. Could you give it a try?

**Comment by arturovt** [comment]
I'm closing it as "implemented". I've added a section to readme about this: https://github.com/ngx-lottie/ngx-lottie#updating-animation

You can open an issue if you face some trouble while trying to use without a doubt :)
